src/DualReader.php

```diff
@@ -78,7 +80,7 @@ public function getPropertyAnnotations(ReflectionProperty $property): array
 
         $attributes = $this->attributeReader->getPropertyAnnotations($property);
 
-        return array_merge($annotations, $attributes);
+        return array_unique(array_merge($annotations, $attributes), SORT_REGULAR);
```

Comment:
Would this keep same annotation that only differ in one parameter?

Reply:
Yes, test for it added.
@see https://github.com/koriym/Koriym.Attributes/pull/14/commits/815149331c9a545366eaf83508b1de1cb67ee80a

Reply:
Awesome 😎👍

Reply:
Will be released tomorrow.😎

Reply:
Thank you! Have a good night sleep 😴